frontend/.dockerignore

```diff
@@ -1,2 +1,3 @@
 README
 Dockerfile
+node_modules
```

Comment:
I don't think it hurts to have it. In case someone happened to build their frontend locally, it could make everything really slow. It's just a standard practice to include it for JS projects.

Reply:
I thought it's standard practice for `.gitignore`, is it for `.dockerignore` too?

Reply:
I see it in most of the dockerized JS projects. It is also mentioned here: https://nodejs.org/de/docs/guides/nodejs-docker-webapp/